README.md

```diff
@@ -1,371 +1,217 @@
-
-## Packages
-
-This repository is a monorepo managed using
-[Lerna](https://github.com/lerna/lerna). This means that we publish
-[many packages](/packages) to NPM from the same codebase.
-
-## Thanks
+<p align="center">
+  <a href="https://next.gatsbyjs.org">
+    <img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" width="60" />
+  </a>
+</p>
+<h1 align="center">
+  Gatsby v2 [beta] · (see <a href="https://github.com/gatsbyjs/gatsby/tree/v1">v1</a>)
+</h1>
+
+<h3 align="center">
+  ⚛️ 📄 :rocket:
+</h3>
+<p align="center">
+  <strong>Blazing fast site generator for React</strong><br>
+  Go beyond static sites: build blogs, ecommerce sites, full-blown apps, and more with Gatsby.
+</p>
+<p align="center">
+  <a href="https://github.com/gatsbyjs/gatsby/blob/master/LICENSE">
+    <img src="https://img.shields.io/badge/license-MIT-blue.svg" alt="Gatsby is released under the MIT license." />
+  </a>
+  <a href="https://travis-ci.org/gatsbyjs/gatsby">
+    <img src="https://travis-ci.org/gatsbyjs/gatsby.svg?branch=master" alt="Current TravisCI build status." />
+  </a>
+  <a href="https://www.npmjs.org/package/gatsby">
+    <img src="https://img.shields.io/npm/v/gatsby.svg?style=flat-square" alt="Current npm package version." />
+  </a>
+  <a href="https://npmcharts.com/compare/gatsby?minimal=true">
+    <img src="https://img.shields.io/npm/dm/gatsby.svg" alt="Downloads per month on npm." />
+  </a>
+  <a href="https://next.gatsbyjs.org/docs/how-to-submit-a-pr/">
+    <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome!" />
+  </a>
+</p>
+
+<h3 align="center">
+  <a href="https://next.gatsbyjs.org/docs/">Quickstart</a>
+  <span> · </span>
+  <a href="https://next.gatsbyjs.org/tutorial/">Tutorial</a>
+  <span> · </span>
+  <a href="https://next.gatsbyjs.org/plugins/">Plugins</a>
+  <span> · </span>
+  <a href="https://next.gatsbyjs.org/docs/gatsby-starters/">Starters</a>
+  <span> · </span>
+  <a href="https://next.gatsbyjs.org/docs/how-to-contribute/">Contribute</a>
+  <span> · </span>
+  Support: <a href="https://spectrum.chat/gatsby-js">Spectrum</a>
+  <span> & </span>
+  <a href="https://discord.gg/0ZcbPKXt5bVoxkfV">Discord</a>
+</h3>
+
+## Table of Contents
+
+-   [Get Up and Running in 5 Minutes](#-get-up-and-running-in-5-minutes)
+-   [Learning Gatsby](#-learning-gatsby)
+-   [Migration Guides](#-migration-guides)
+-   [For Advanced Developers: Code Samples & Recipes](#-code-samples-and-recipes)
+-   [How to Contribute](#-how-to-contribute)
+-   [Thanks to Our Contributors and Sponsors](#-thanks-to-our-contributors-and-sponsors)
+
+
+## 🚀 Get Up and Running in 5 Minutes
+
+You can get a new Gatsby site up and running on your local dev environment in 5 minutes with these three steps:
+
+1.  **Create a Gatsby site from a Gatsby starter.**
+
+    Get your Gatsby blog set up in a single command:
+
+    ```sh
+    npx gatsby-cli new my-blazing-fast-site https://github.com/gatsbyjs/gatsby-starter-default#v2
```

Comment:
Also until we release v2, let's keep pushing people to use the default version.

Reply:
I don't think this should be cherry-picked to the `v1` branch. Instead, this should stay only on the `master` branch and content within should remain relevant to v2. This makes the most sense to me since v2 is already the `master` branch of the repository.